php-checks/src/main/java/org/sonar/php/checks/OpeningPHPTagCheck.java

```diff
@@ -60,19 +59,14 @@ public void visitNode(AstNode astNode) {
    * The node contains everything before the first opening include HTML if present
    * this allows to ensure reporting the issue on the correct line.
    */
-  private int getLineToReport(AstNode node) {
-    return node.getTokenValue().split("(?:\r)?\n|\r").length;
+  private int getLineToReport(String openingTag) {
+    return openingTag.split("(?:\r)?\n|\r").length;
   }
 
-  private boolean isNotAuthorisedTag(AstNode node) {
-    int offset = 1 + LONG_TAG.length();
-    String openingTag = node.getTokenOriginalValue();
-
-    if (openingTag.length() > offset) {
-      openingTag = openingTag.substring(offset);
-    }
-
-    return !openingTag.contains(LONG_TAG) && !openingTag.contains(SHORT_ECHO_TAG) && openingTag.contains(SHORT_TAG);
+  private boolean isAuthorisedTag(String openingTagWithContentBefore) {
+      return openingTagWithContentBefore.endsWith(LONG_TAG)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AIndentationCheck)